Face/Microsoft.Azure.CognitiveServices.Samples.Face/VerifyFaceToFace.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Microsoft.Azure.CognitiveServices.Samples.Face
+{
+    using Microsoft.Azure.CognitiveServices.Vision.Face;
+    using Microsoft.Azure.CognitiveServices.Vision.Face.Models;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading.Tasks;
+
+    public static class VerifyFaceToFace
```

Comment:
We should remove this since this is included in the Verify PR?

Reply:
Sure